tests/zerobase/host/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../zerobase.edl)
+
+add_custom_command(
+  OUTPUT zerobase_u.h zerobase_u.c
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND
+    edger8r --untrusted ${EDL_FILE} --search-path ${PROJECT_SOURCE_DIR}/include
+    ${DEFINE_OE_SGX} --search-path ${CMAKE_CURRENT_SOURCE_DIR})
+
+add_executable(zerobase_host host.cpp zerobase_u.c)
+
+target_include_directories(zerobase_host PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
+                                                 ${CMAKE_CURRENT_SOURCE_DIR})
+
+# TODO: Need to find a way around hard-coding th host application base address
+# on the application CMakeList.txt.
```

Comment:
Typo "th".  Also if this is a TODO, please file an issue to track it, and list the issue # in the TODO comment.